### ISSUE TITLE
SimpleMap locations title appears as SimpleMap in AIOSEOP

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/simplemap#162]
During testing I found that AIOSEOP refers to SimpleMap Locations under Custom Post Type Settings and XML Sitemap as SimpleMap.

<img width="1411" alt="screen shot 2017-02-09 at 3 22 01 pm" src="https://cloud.githubusercontent.com/assets/17548525/22801622/9530c5ee-eedb-11e6-9c48-4b339b9cb0fe.png">

**Comment by wpsmort** [comment]
This was because where we register the CPT in classes/locations.php we set the label for name as SimpleMap and didn't set a menu_name.

			'labels'               => array(
				'name'               => 'SimpleMap',
				'singular_name'      => 'Location',
				'add_new_item'       => 'Add New Location',
				'all_items'         => 'Locations',
				'edit_item'          => 'Edit Location',
				'new_item'           => 'New Location',
				'view_item'          => 'View Locations',
				'search_items'       => 'Search Locations',
				'not_found'          => 'No Locations found',
				'not_found_in_trash' => 'No Locations found in trash',
			),

**Comment by wpsmort** [comment]
@michaeltorbert I have changed the name label from SimpleMap to SimpleMap Locations so that it improves how locations are displayed in other plugins that reference the CPT name (such as AIOSEOP).  

I've tested and can see no negative impact from this change and I don't believe that we'd be referencing the CPT name label anywhere else in the plugin.  I wanted to check with you first before we close this issue.
